### PR TITLE
correct ReactContext.js if block writting at line21

### PR DESCRIPTION
**Pull request by fantasticsoul**
correct ReactContext.js if block writting at line21

### CODE REVIEW

File: packages/react/src/ReactContext.js [hunk 1]
```diff
@@ -20,16 +20,15 @@ export function createContext<T>(
...
-          typeof calculateChangedBits === 'function',
-        'createContext: Expected the optional second argument to be a ' +
-          'function. Instead received: %s',
-        calculateChangedBits,
-      );
-    }
+  }
+  if (__DEV__) {
+    warningWithoutStack(
+      calculateChangedBits === null ||
```
Comment (audiolion) [inline packages/react/src/ReactContext.js]: this change could cause this warning to be thrown now where it wouldn't before, `calculateChangedBits` is `undefined`, then gets set to `null`, then you are in `__DEV__` and `calculateChangedBits === null` now passes.

Comment (acdlite) [reply packages/react/src/ReactContext.js]: `warningWithoutStack` fires when its first argument is falsey. When `calculateChangedBits` is `null`, it does not warn.

So in either version, if `calculateChangedBits` is undefined, there's no warning. So the behavior is the same.

**Comment by acdlite** [comment]
Thanks for the PR, but the behavior is the same either way. Explanation: https://github.com/facebook/react/pull/16285#discussion_r311183195

File: packages/react/src/ReactContext.js (hunk 1 above)
Comment (audiolion) [reply packages/react/src/ReactContext.js]: ah I assumed it called when `true`, my bad.

**Pull request closed without merging**